design/search.md

```diff
@@ -0,0 +1,300 @@
+
+
+# Federated Search
+![VPCzJyCm48Pt_ufJftP0x1bGAuIgIhG22XDYEE9hQicnW-qW-FVu8qT2HTlfSkzpxtrONVg0BlIj5bW7ww3yNZnnY3v_YIvYgbOTcW2pbNDe6d8LR56PMOHoS0wwjUQWceoLy1ou9tJrCOCbl0oEninVjjzPIJ1trDf0YrILCqA8lE_LJLu2edkw2N1Tr7C-wiM-WYVwwCa7gFCt79GcKRI_5Ce1yV2h3stuAcpEsOrH (1)](https://github.com/overture-stack/arranger/assets/1486054/87bb0689-38c6-49eb-ab65-96696a04d86a)
+
+## Prerequisites
+- Local data must be configured
+- ENV flag passed into process
+- "search" config file present
+  
+## Config
+- Read "search config" from file
+- Export object containing node data for use in gql endpoint
+
+Example Config provided:
+
+| Field         | Type   | Note                                                                                                                            |
+| ------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| endpoint      | string | Full url to gql endpoint                                                                                                        |
+| arrangerField | string | Endpoints can have other data on the root object. Need to get specific Arranger config object. Appears to be "file" by default. |
+
+Example Search Node data:
+
+| Field  | Description                 |
+| ------ | --------------------------- |
+| url    | gql endpoint                |
+| name   | name of node eg. Toronto    |
```

Comment:
Seeing this here, we may want to include a `name` property to our node config. Arranger nodes aren't currently reporting a name of their own, so this would help us label each node in our responses.